plat/unix/update_tags.sh

```diff
@@ -99,7 +99,12 @@ if [ -f "$TAGS_FILE" ]; then
     if [ "$UPDATED_SOURCE" != "" ]; then
         echo "Removing references to: $UPDATED_SOURCE"
         tab="	"
-        cmd="grep --text -Ev '^[^$tab]+$tab$UPDATED_SOURCE$tab' '$TAGS_FILE' > '$TAGS_FILE.temp'"
+        # use ripgrep if available
+        if which rg; then
+            cmd="rg --text -ev '^[^$tab]+$tab$UPDATED_SOURCE$tab' '$TAGS_FILE' > '$TAGS_FILE.temp'"
+        else 
+            cmd="grep --text -Ev '^[^$tab]+$tab$UPDATED_SOURCE$tab' '$TAGS_FILE' > '$TAGS_FILE.temp'"
+        fi
```

Comment:
I have tested this `ripgrep` command with a real call of vim-gutentags:

```sh
grep --text -Ev '^[^	]+	sxcs.c	' 'tags-c' > 'tags-c.temp'
```

The `rg` version of the command (`rg --text -ev`) gives me a different output plus an error:

```
^[^     ]+      sxcs.c  : No such file or directory (os error 2)
```

I suggest the following changes:

1. Drop `-e` from the `rg` command. (`-ve` as used in the PR description works, too.)
2. Add `--no-config`. (Otherwise a setting in `~/.config/ripgrep/ripgreprc` might lead to lines ending in `[... omitted end of long line]`.)
3. Use `command -v` instead of `which` because it's faster.
4. Use a `GREP` variable for the variable part of the command as requested by @ludovicchabant.

```suggestion
        if command -v rg > /dev/null 2>&1 ; then
            GREP='rg --no-config'
        else
            GREP='grep -E'
        fi
        cmd="$GREP --text -v '^[^$tab]+$tab$UPDATED_SOURCE$tab' '$TAGS_FILE' > '$TAGS_FILE.temp'"
```